code/plugins/authorizenet/authorizenet/tmpl/default.php

```diff
@@ -91,15 +91,15 @@ function plg_auth_showHide()
 						<?php echo JText::_('FIRST_NAME') . JText::_('PLG_PAYMENT_AUTHORIZENET_REQUIRED_MARK'); ?>
 					</label>
 					<div class="controls">
-						<input class="inputbox required" id="cardfname" type="text" name="cardfname" size="35" value="<?php echo !empty($userInfo['firstname']) ?$userInfo['firstname']:'';?>" />
+						<input class="inputbox required" id="cardfname" type="text" name="cardfname" size="35" value="<?php echo !empty($userInfo['firstname']) ?$userInfo['firstname']:'';?>" pattern="[a-zA-Z0-9\s\.]+"/>
```

Comment:
@deepgawade89 In french the name can be - Melodene D'Souza . and this should get accepted